parmed/amber/netcdffiles.py

```diff
@@ -262,14 +262,18 @@ def coordinates(self, stuff):
 
     @property
     def velocities(self):
-        vels = self._ncfile.variables['velocities'][:]
-        return (vels.reshape((-1, self.atom, 3)) * self.velocity_scale)
+        if self.hasvels:
+            vels = self._ncfile.variables['velocities'][:]
+            return (vels.reshape((-1, self.atom, 3)) * self.velocity_scale)
+        else:
+            return None
 
     @velocities.setter
     def velocities(self, stuff):
         self._ncfile.variables['velocities'][:] = \
                     np.reshape(stuff, (self.atom, 3)) / self.velocity_scale
         self.flush()
+        self.hasvels = True
```

Comment:
The `velocities` setter here only gets called when *writing* NetCDF files.  So if you're reading NetCDF files, `hasvels` will *always* be `False`, even if it actually has velocities.